Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.6.12-buster AS compiler
+LABEL maintainer="adrian.w.roque@gmail.com"
+
+WORKDIR /usr/src/app
+ENV PATH=/root/.local/bin:$PATH
+
+ARG CHROMIUM_VERSION=83.0.4103.116-1~deb10u3
+ARG CHROMEDRIVER_VERSION=83.0.4103.39
+RUN apt-get update && apt-get install -y chromium=$CHROMIUM_VERSION
+RUN curl -LO https://chromedriver.storage.googleapis.com/$CHROMEDRIVER_VERSION/chromedriver_linux64.zip && \
+    unzip chromedriver_linux64.zip -d /usr/lib/chromium-browser						&& \
+    ln -s /usr/lib/chromium-browser/chromedriver /usr/local/bin
+
+RUN curl https://raw.githubusercontent.com/AdrianWR/urlbuilder/master/urlbuilder/urlbuilder.py > urlbuilder.py
```

Comment:
can we `curl` Adrian's code from this repo?

Reply:
I think you should be able to, if you set up github credentials on the VM prior to building the image. However, I think it's better to include this file in the master branch and then just `COPY` it instead of `RUN curl`. That way credentials are not a problem, and the code is self-contained.